unblob/cli.py

```diff
@@ -54,5 +55,21 @@ def main():
         cli.invoke(ctx)
 
 
+def main():
+    try:
+        run()
+    except SystemExit as e:
+        EXIT_CODE.set(e.code)
+    except Exception:
+        EXIT_CODE.set(1)
+        logger.exception("Unhandled exception during unblob")
+    finally:
+        default_exit_code = 0
+        exit_code = EXIT_CODE.get(default_exit_code)
+        logger_func = logger.info if exit_code == default_exit_code else logger.error
+        logger_func(f"Exiting with code: {exit_code}")
```

Comment:
We don't need to log this at all. CI will fail, Docker container will show the exit status, shell sets the `$?` variable, so it's pretty much redundant. The code in this except block would be very simple if you don't do this.